ios/ElectrodeReactNativeBridge/ElectrodeBridgeProtocols.h

```diff
@@ -79,12 +79,19 @@ typedef void (^ElectrodeBridgeEventListener)(id _Nullable eventPayload);
 /**
  * Register the request handler
  * @param name name of the request
+ * @param uuid uuid of the request handler
  * @param completion call back to be issued for a given request.
  */
-- (NSUUID *)registerRequestCompletionHandlerWithName:(NSString *)name
-                                   completionHandler:
-                                       (ElectrodeBridgeRequestCompletionHandler)
-                                           completion;
+- (void) registerRequestCompletionHandlerWithName:(NSString *)name
```

Comment:
OCD comment: seems like you have no coding convention for whitespace? Within 9 rows I can see "(NSString *)name", "<space>(NSString *)<space>uuid" and "<space>(NSUUID *)uuid" :) Can't remember any more how we automated this kind of things, but should be possible.

Reply:
Wondering, if e.g. name should be _Nonnull